backend/src/LibBackend/SqlCompiler.fs

```diff
@@ -164,13 +164,33 @@ let rec inline'
   : Expr =
   RuntimeTypesAst.postTraversal
     (fun expr ->
+
+      let rec mapLetPattern
+        (symtable : Map<string, Expr>)
+        (currentExpr : Expr, letPattern : LetPattern)
+        : Map<string, Expr> =
+        match letPattern with
+        | LPVariable (_id, name) -> Map.add name currentExpr symtable
+        | LPTuple (_id, first, second, theRest) ->
+          match currentExpr with
+          | ETuple (_, firstExpr, secondExpr, restExpr) ->
+            // Each expression is paired with the corresponding let pattern
+            let zipped =
+              List.zip
```

Comment:
We should check length before List.zip as it will generate the built-in error from List.zip and we'd prefer to have one of our own errors (`Exception.raiseCode`)

Reply:
done